lib/pcr.c

```diff
@@ -177,6 +178,59 @@ static bool pcr_unset_pcr_sections(TPML_PCR_SELECTION *s) {
     return true;
 }
 
+bool pcr_print_pcr_struct_le(TPML_PCR_SELECTION *pcr_select, tpm2_pcrs *pcrs) {
+
+    UINT32 vi = 0, di = 0, i;
+    bool result = true;
+
+    tpm2_tool_output("pcrs:\n");
+
+    // Loop through all PCR/hash banks
```

Comment:
line comments to be used at end of line.
```suggestion
    /* Loop through all PCR/hash banks */
```